libkbfs/conflict_resolver.go

```diff
@@ -2873,6 +2895,12 @@ func (cr *ConflictResolver) syncBlocks(ctx context.Context, lState *lockState,
 		}
 	}
 
+	defer func() {
```

Comment:
`syncTree` doesn't actually put any blocks -- the blocks don't need any cleanup until the `doBlockPuts` call just below this.